CHANGELOG.next.asciidoc

```diff
@@ -24,6 +24,8 @@ you can achieve this by overwriting the value using an `add_fields` processor. {
 
 *Heartbeat*
 
+- Adjust State loader to only retry for failed requests and not for 4xx. {pull}37981[37981]
```

Comment:
Is this a breaking change or bugfix?

Reply:
I'm OK with this being a bugfix, I just meant to say it's listed under **Breaking changes**

Reply:
oh I see, thanks 🚀 , will correct at once